api/src/main/java/org/openmrs/module/registrationapp/model/RegisterPersonRelationshipWidget.java

```diff
@@ -0,0 +1,79 @@
+package org.openmrs.module.registrationapp.model;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+
+public class RegisterPersonRelationshipWidget extends Widget{
```

Comment:
Nitpick, but if you can run a formatter on this file to clean up the spacing before/after curly braces, between methods, between properties, etc that would be good.  It seems a bit inconsistent.